debezium/src/test/java/org/apache/camel/example/debezium/DebeziumTest.java

```diff
@@ -80,11 +78,7 @@ class DebeziumTest extends CamelMainTestSupport {
                 .withPassword(SOURCE_DB_PASSWORD)
     );
     @RegisterExtension
-    private static final CassandraService CASSANDRA_SERVICE = new CassandraLocalContainerService(
-        new CassandraContainer<>(CASSANDRA_IMAGE)
```

Comment:
we lost the infra image information but the test seems to be ok anyway, see https://github.com/apache/camel-examples/pull/143#issuecomment-2800686093 where the debezium module is no tlisted